src/frontend/src/styles/main.css

```diff
@@ -329,8 +329,14 @@
   --rs-split-page-padding--mobile: calc(var(--vs-stack) * 3);
   --rs-split-page-left-pane-padding: calc(var(--vs-gutter) * 8);
   --rs-split-page-right-pane-padding--desktop: 10vmax;
-  --rs-split-page-right-pane-padding--mobile: calc(var(--vs-stack) * 2);
+  --rs-split-page-right-pane-padding--mobile--vertical: calc(
```

Comment:
my preference would be: `--rs-split-page-right-pane-padding-x--mobile` double modifiers in names a bit weird inho

Reply:
I notice now that the variables names say "padding". I usually try to avoid technical implementation in the var name, because it limits reuse. I usually use:

- `somethin-stack` for spaces above or below outside of the element
- `something-inline` for spaces left or right outside of the element
- `something-bezzel` or `gutter` for spaces withing (or bezzel-x bezzel-y) 

https://codepen.io/meodai/pen/zNaEyM

But feel free to change this an other time if at all since its a bit out of scope for this PR. And I did not notice last time

Reply:
True, I renamed all the new vars to use the convention. Makes sense to me.